dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1583,7 +1651,7 @@ def is_int(x):
             return a == b
 
     @staticmethod
-    def np_type_to_type(val):
+    def np_type_to_type(val: Any) -> Union[int, float]:
```

Comment:
Just want to validate this is correct as the value may be an `np.float`. Additionally, the case for `np.int`
https://numpy.org/devdocs/reference/typing.html

Reply:
The function now checks if `val` is `np.floating` or `np.integer` which should cover everything.

Reply:
wonder if this should be `Union[int, float]`

Reply:
The function is used in places where the type is neither `np.floating` or `np.integer`, and `val` is returned in this case. For example, line 305 has `np_type_to_type(self.mode)` and `self.mode` is `List[float]`.

Reply:
cool cool ... let's hold off on this then for now `Any` will work

Reply:
oof, that's weird.